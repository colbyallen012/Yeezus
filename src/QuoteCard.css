.pablo {
  height: 17%;
  width: 17%;
}

h2 {
  font-size: 90px;
}

p {
  font-size: 40px;
}

