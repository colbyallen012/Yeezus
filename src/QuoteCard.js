import React, { Component } from 'react';
import pablo from './Utils/pablo.png';
import './QuoteCard.css';

export class QuoteCard extends Component {

  newQuote = async (e) => {
    e.preventDefault()
    this.props.getQuote()
  }

  time = () => {
   var currTime = new Date();

   return currTime.toLocaleString('en-US', { 
     hour: 'numeric', 
     minute: 'numeric', 
     hour12: true 
    })
  }

  render() {
    return (
      <div className='QuoteCard'>
        <h2>{this.time()}</h2>
        <p className='quote'>{this.props.quote}</p>
        <img className='pablo' src={pablo} alt='kanye-west'/>
        <form onSubmit={this.newQuote}>
          <button className='quote-button'>New Quote</button>
        </form>
      </div>
    )
  }
}

export default QuoteCard;