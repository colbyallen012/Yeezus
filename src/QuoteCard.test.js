import React from 'react';
import { shallow } from 'enzyme';
import QuoteCard from './QuoteCard';

describe('QuoteCard', () => {
  let wrapper, instance;

  beforeEach(() => {
    wrapper = shallow(<QuoteCard />)
    instance = wrapper.instance()
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})