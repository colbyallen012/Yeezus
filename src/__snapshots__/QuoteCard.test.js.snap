// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteCard should match snapshot 1`] = `ShallowWrapper {}`;
